PowerEditor/src/WinControls/OpenSaveFileDialog/CustomFileDialog.cpp

```diff
@@ -902,6 +927,8 @@ class CustomFileDialog::Impl
 	bool _enableFileTypeCheckbox = false;
 	bool _fileTypeCheckboxValue = false;	// initial value
 	wstring _fileTypeCheckboxLabel;
+	bool _isSavingAsCopy = false;
+	bool _isOpeningTheCopy = false;
```

Comment:
IMO, for this feature we need only 1 member variable than 2 bool _isSavingAsCopy & _isOpeningTheCopy, for the reason to make the class class `CustomFileDialog::Impl` simpler.
The 2 bool member variables can be reduced indeed to 1 unsigned char member variable, for 1 feature.
```cpp
unsigned char _savingAsCopyInfo = 0;
```

and use mask SAVE_AS_COPY_DLG (value 0x02) and mask SAVE_AS_COPY_OPEN (value 0x01) to retrieve these 2 booleans when it's necessary.

Reply:
OK, I will do it when I return from my most recent trip (extends through 21 July).

Reply:
Bon voyage @alankilborn !